react/features/connection-indicator/components/AbstractConnectionIndicator.js

```diff
@@ -85,8 +95,13 @@ class AbstractConnectionIndicator<P: Props, S: State> extends Component<P, S> {
      * returns {void}
      */
     componentDidMount() {
-        statsEmitter.subscribeToClientStats(
-            this.props.participantId, this._onStatsUpdated);
+        if (this.props._sourceNameSignalingEnabled) {
+            statsEmitter.subscribeToClientStats(
+                this.props._sourceName, this._onStatsUpdated);
```

Comment:
Will this work since only the resolution/fps stats are indexed by source-name while all the other stats are indexed by participant id ? Doesn't  the callback func `_onStatsUpdated` needs to be updated for the case when source-name signaling is enabled ?